Mathlib/Data/Finset/Image.lean

```diff
@@ -162,6 +162,18 @@ theorem map_subset_map {s₁ s₂ : Finset α} : s₁.map f ⊆ s₂.map f ↔ s
    fun h => by simp [subset_def, Multiset.map_subset_map h]⟩
 #align finset.map_subset_map Finset.map_subset_map
 
+/-- The `Finset` version of `Equiv.subset_symm_image_iff`. -/
+theorem subset_symm_map_iff {t : Finset β} {f : α ≃ β} : s ⊆ t.map f.symm ↔ s.map f ⊆ t := by
```

Comment:
I understand why you had it in the order you did (for parallelism with `Equiv` versions), but I think this makes more sense here because we don't have any infix notation for `Finset.map`. And I don't think the `_iff` is necessary.
```suggestion
theorem subset_map_symm {t : Finset β} {f : α ≃ β} : s ⊆ t.map f.symm ↔ s.map f ⊆ t := by
```

Reply:
OK, done!